docs/skopeo.1.md

```diff
@@ -64,6 +64,11 @@ Uses the system's trust policy to validate images, rejects images not trusted by
 
   _destination-image_ use the "image name" format described above
 
+  **--authfile** _path_
+
+  Path of the authentication file. Default is ${XDG_RUNTIME_DIR}/containers/auth.json
+  which is set using `kpod login`
+
```

Comment:
Just double checking as I've not tried.  If you use the $HOME/.docker/config.json file as the auth file, is docker happy afterwards?  I.e. if you do 'kpod login --auth ~/.docker/config.json', can you then run a docker command that requires auth without it spitting up?  Really is not related to this review though.

Reply:
I actually haven't tried that yet. It might not work because there is a slight difference in the way kpod and docker store the registry name in auth.json and config.json.